common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -91,6 +96,13 @@
     private static InteractiveTokenCommand sCommand = null;
     private static final CommandResultCache sCommandResultCache = new CommandResultCache();
 
+    private static final TreeSet<String> nonCacheableErrorCodes = new TreeSet(
```

Comment:
Any reason we need to use a `TreeSet` vs a `HashSet`? Is there any ordering dependency?

Reply:
For such a small number of items, TreeSet can typically be (slightly) more efficient.